src/libs/antares/study/scenario-builder/rules.cpp

```diff
@@ -201,6 +205,33 @@ namespace ScenarioBuilder
 		}
 	}
 
+	void Rules::sendWarningsForDisabledClusters()
+	{
+		map < string, vector<uint> >::iterator it = disabledClustersOnRuleActive.begin();
+		for (; it != disabledClustersOnRuleActive.end(); it++)
```

Comment:
peut éventuellement être raccourci :
`for (auto it = disabledClustersOnRuleActive.begin(); it != disabledClustersOnRuleActive.end(); it++)`